libp2p/switch.nim

```diff
@@ -191,7 +191,7 @@ proc upgradeIncoming(s: Switch, conn: Connection) {.async, gcsafe.} =
                        {.async, gcsafe, closure.} =
     trace "Securing connection"
     let secure = s.secureManagers[proto]
-    let sconn = await secure.secure(conn)
+    let sconn = await secure.secure(conn, false)
```

Comment:
we should keep using the default here.

Reply:
Any reason why being explicit is bad here tho?
Helps anyone reading the code imho

Reply:
Nothing against it, I'm just concerned we're changing the intent of the API and there is no reasoning for it anywhere. Granted that the flag was introduced in prior commits, but I can't find anything that explains the change properly.